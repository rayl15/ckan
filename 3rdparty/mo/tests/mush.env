USER=jwerle
GENDER=male
THING=apple
COLOR=red
PERSON=tobi
ADJECTIVE=cool
