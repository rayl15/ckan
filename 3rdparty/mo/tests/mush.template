{{! this is a comment }}

{{USER}} is {{GENDER}}
{{THING}} is {{COLOR}}
{{PERSON}} is {{ADJECTIVE}}
{{USER}} is friends with {{PERSON}}
{{var}} {{value}}
