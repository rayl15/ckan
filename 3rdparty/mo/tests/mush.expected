
jwerle is male
apple is red
tobi is cool
jwerle is friends with tobi
 
